src/pages/index.js

```diff
@@ -25,37 +26,115 @@ editProfileValidator.enableValidation();
 const addCardValidator = new FormValidator(config, newCardEditForm);
 addCardValidator.enableValidation();
 
+const avatarFormValidator = new FormValidator(config, avatarForm);
+avatarFormValidator.enableValidation();
+
+const removeCardValidator = new FormValidator(config, removeCardForm);
```

Comment:
You don't need to validate the confirmation form because it doesn't contain inputs